tools/js_optimizer.py

```diff
@@ -61,13 +61,15 @@ def get_native_optimizer():
     sys.exit(1)
 
   # Allow users to override the location of the optimizer executable by setting an environment variable EMSCRIPTEN_NATIVE_OPTIMIZER=/path/to/optimizer(.exe)
-  if os.environ.get('EMSCRIPTEN_NATIVE_OPTIMIZER') and len(os.environ.get('EMSCRIPTEN_NATIVE_OPTIMIZER')) > 0:
-    logging.debug('env forcing native optimizer at ' + os.environ.get('EMSCRIPTEN_NATIVE_OPTIMIZER'))
-    return os.environ.get('EMSCRIPTEN_NATIVE_OPTIMIZER')
+  opt = os.environ.get('EMSCRIPTEN_NATIVE_OPTIMIZER')
+  if len(opt):
+    logging.debug('env forcing native optimizer at ' + opt))
+    return opt
   # Also, allow specifying the location of the optimizer in .emscripten configuration file under EMSCRIPTEN_NATIVE_OPTIMIZER='/path/to/optimizer'
-  if hasattr(shared, 'EMSCRIPTEN_NATIVE_OPTIMIZER') and len(shared.EMSCRIPTEN_NATIVE_OPTIMIZER) > 0:
-    logging.debug('config forcing native optimizer at ' + shared.EMSCRIPTEN_NATIVE_OPTIMIZER)
-    return shared.EMSCRIPTEN_NATIVE_OPTIMIZER
+  opt = getattr(shared, 'EMSCRIPTEN_NATIVE_OPTIMIZER')
+  if len(opt):
```

Comment:
Does `getattr` return `None` if it doesn't have it? Cause `len(None)` fails
This should just work as `if opt:` though